### DISCUSSION TITLE
Releases of iptv-org/epg repo?

### DISCUSSION THREAD

**Discussion by nt74** [iptv-org/community#517] [Ideas]
I have just made an AUR package with a custom bash script (that uses the npm package and run script) and solution for the Arch Linux including information on their ArchWiki pages.
The AUR package is calling on git commits from the github master upstream. This upstream gets a lot of edits etc.
Logically I can understand that creating Releases isn't that optimal for such a project like this with a lot of revisions and edits almost daily.
But is there any possibility to create Releases with larger chunks of edits/revisions etc, so that people (like me) that wants to create packages can easier lock their packages in accordance with the epg master releases upstream?

**Comment by freearhey** [comment]
I just created the first release: https://github.com/iptv-org/epg/releases/tag/2023.12.1. 

Hope this helps!

**Comment by nt74** [comment]
Many thanks @freearhey it's much better to lock to latest Release for a package.
